core/kernel/autoload_spec.rb

```diff
@@ -60,6 +60,32 @@ def check_autoload(const)
       ruby_exe(fixture(__FILE__, "autoload_frozen.rb")).should == "#{frozen_error_class} - nil"
     end
   end
+
+  describe "when called from included module's method" do
+    module KSAutoloadMethod
+      def setup_autoload
+        autoload :KSAutoloadFromIncludedModule, fixture(__FILE__, "autoload_from_included_module.rb")
+      end
+    end
+    class KSAutoloadMethodIncluder
+      include KSAutoloadMethod
+    end
+    KSAutoloadMethodIncluder.new.setup_autoload
```

Comment:
Fixtures should be in classes.rb and the setup only be called in a `before` block, so that. e.g. this file can be loaded and the rest of the specs be run even if autoload would expect a Module as the receiver. I fixed it in 273ad6ffa0b04893f5297ad53f4581ec5e037e05.